_data/glossary/kubelet.yaml

```diff
@@ -0,0 +1,10 @@
+id: kubelet
+name: Kubelet
+full-link: docs/reference/generated/kubelet
+tags:
+- fundamental
+- core-object
+short-description: >
+  An agent that runs on each node.
```

Comment:
This part is hard -- solid short descriptions. I appreciate how short you've made it, but perhaps it's a bit too short. Remember that the short description appears in some places without any of the long description information, so can you find a way to incorporate a little more information about what kind of agent it is? Maybe this section could help? https://kubernetes.io/docs/concepts/overview/components/#kubelet

Reply:
@Bradamant3  sounds good. Thanks!